plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/dialogs/SuggestionInformationControl.java

```diff
@@ -0,0 +1,239 @@
+                           String propertyString = DBInfoUtils.getPropertyString(collector, descriptor);
+                           if (CommonUtils.isEmpty(propertyString)) {
+                               continue;
+                           }
+                           Composite placeholder = UIUtils.createPlaceholder(metadataComposite, 2);
+                           placeholder.setLayoutData(new GridData(GridData.FILL_HORIZONTAL));
+                           Label label = new Label(placeholder, SWT.READ_ONLY);
+                           label.setText(descriptor.getDisplayName() + ":");
+                           label.setFont(boldFont);
+                           Text valueText = new Text(placeholder, SWT.READ_ONLY);
+                           valueText.setText(propertyString);
+                       }
+                       infoComposite.layout(true, true);
+                       mainComposite.layout(true, true);
+                   });
+                }
+            }
+        });
+        resolveObject.schedule();
+    }
+
+    private void createTreeControl(JDBCTable<?, ?> input) {
+        GridData gridData = new GridData(GridData.FILL_BOTH);
+        gridData.horizontalSpan = 0;
+        gridData.verticalSpan = 0;
+        IEditorSite subSite = new SubEditorSite(UIUtils.getActiveWorkbenchWindow()
+            .getActivePage()
+            .getActivePart()
+            .getSite());
+        DBNDatabaseNode node = DBWorkbench.getPlatform().getNavigatorModel().findNode(input);
+        ItemListControl itemListControl = new ItemListControl(
+            tableComposite,
+            SWT.NONE,
+            subSite,
+            node,
+            null
+        );
+        itemListControl.setLayoutData(gridData);
+        final Object[] columnNodes = new Object[1];
+        AbstractJob abstractJob = new AbstractJob("Populating table tip") {
+            @Override
+            protected IStatus run(DBRProgressMonitor monitor) {
+                monitor.beginTask("load table columns", 1);
+                try {
+                    columnNodes[0] = getColumnNodes(monitor, node);
+                } catch (DBException e) {
+                    log.error("Error reading table columns", e);
+                    return Status.CANCEL_STATUS;
+                } finally {
+                    monitor.done();
+                }
+                return Status.OK_STATUS;
+            }
+        };
+        abstractJob.addJobChangeListener(new JobChangeAdapter() {
+            @Override
+            public void done(IJobChangeEvent event) {
+                UIUtils.syncExec(() -> {
+                    itemListControl.appendListData((Collection<DBNNode>) columnNodes[0]);
+                    tableComposite.layout(true, true);
+                });
+            }
+        });
+        abstractJob.schedule();
+    }
+
+    @NotNull
+    private Collection<DBNNode> getColumnNodes(
+        DBRProgressMonitor monitor, @Nullable DBNNode node
```

Comment:
Like talking to a wall.

```suggestion
        @NotNull DBRProgressMonitor monitor,
        @Nullable DBNNode node
```